client-tools/Export/ExportFeature.cpp

```diff
@@ -98,11 +98,23 @@ void ExportFeature::collectOptions(std::shared_ptr<options::ProgramOptions> opti
       "restrict to collection name (can be specified multiple times)",
       new VectorParameter<StringParameter>(&_collections));
 
-  options->addOption("--query", "AQL query to run", new StringParameter(&_query));
-  
-  options->addOption("--query-max-runtime", "runtime threshold for AQL queries (in seconds, 0 = no limit)", 
-                     new DoubleParameter(&_queryMaxRuntime))
-                     .setIntroducedIn(30800);
+  options->addOldOption("--query", "custom-query");
+  options
+      ->addOption("--custom-query", "AQL query to run", new StringParameter(&_customQuery))
+      .setIntroducedIn(30800);
```

Comment:
I don't think that `--query` (or `--custom-query`) was introduced with 3.8.0. The previous version of the code didn't have a `.setIntroducedIn` attribute for this startup option.

Reply:
Addressed